### DISCUSSION TITLE
Welcome to MifareClassicTool Discussions!

### DISCUSSION THREAD

**Discussion by ikarus23** [ikarus23/MifareClassicTool#413] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of the MCT community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by Hasanov07** [comment]
Hello, how can I get a checksum for the mifari classic 1k card 
UID: 045B277A943480
45FE3004D555AE300108F4F9A8560002
000000000000000000000000A2DF7DAA
000000000000000000000000A2DF7DAA
AE3D65A3DAD4787788000F1C63013DBA

A2DF7DAA checksum How many times have I tried to get it right, but I couldn't. If the checksum is not correct, the card does not work, I need to calculate the checksum correctly so that the card works, please help me, thank you very much in advance

**Reply by ikarus23** [reply]
Hi. Next time, please open a new discussion and not use this common introduction.

Regarding to your question: Looks like a common CRC32 with some custom polynomial/initial value/final xor. But it is hard to tell if all input data are `0000...`. Do you have other samples where the data is not zeros?

**Reply by Hasanov07** [reply]
> Hi. Next time, please open a new discussion and not use this common introduction.
> 
> 
> 
> Regarding to your question: Looks like a common CRC32 with some custom polynomial/initial value/final xor. But it is hard to tell if all input data are `0000...`. Do you have other samples where the data is not zeros? 

No, it's not zero, it's a regular Mythari classic card.
